docs/getting_started/intellij_plugin.md

```diff
@@ -0,0 +1,96 @@
+This plugin provides support for working with Chutney scenarios files in IntelliJ IDEA based IDEs.
+# Installation
+
+* Download chutney-idea-plugin-X.Y.zip from [available releases](https://github.com/chutney-testing/chutney-idea-plugin/releases){:target=_blank}.
+* In your Intellij Ide, go to File -> Settings -> Plugins.
+* Click install plugin from disc and select the downloaded zip.
+  ![plugin installation](../img/idea_plugin/install-plugin.gif)
+* Restart your IDE.
+
+
+
+# Configuration
+Configuration files can be found in the [intellij_config_folder](https://intellij-support.jetbrains.com/hc/en-us/articles/206544519-Directories-used-by-the-IDE-to-store-settings-caches-plugins-and-logs#:~:text=Configuration%20(idea.config.path)%3A%20%25APPDATA%25%5CJetBrains%5CIntelliJIdea2022.2){:target=_blank}`/chutney-idea-plugin/conf` folder.
+### Define an environment
```

Comment:
No point of having a single sub-menu here.
As the environment file is mandatory and could be opened by menu, what's the point of giving its path ?
Maybe, just indicate how to access it via menu.

Reply:
The environment is not created by default. It must be created manually. So user must know where to create it.